content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,421 @@
+    Nonbonded:     		None
+    vdW:           		43.8144296 kJ / mol
+    Electrostatics:		-707.0098095999999 kJ / mol
+
+```python
+print(get_openmm_energies(out))
+```
+    Energies:
+
+    Bond:          		17.309860229492188 kJ / mol
+    Angle:         		195.29534912109375 kJ / mol
+    Torsion:       		13.520315170288086 kJ / mol
+    Nonbonded:     		None
+    vdW:           		43.807859356853584 kJ / mol
+    Electrostatics:		-707.037225348007 kJ / mol
+
+```python3
+print(get_lammps_energies(out))
+```
+    Energies:
+
+    Bond:          		4.1371621 kcal / mol
+    Angle:         		46.67671 kcal / mol
+    Torsion:       		3.2314347718102 kcal / mol
+    Nonbonded:     		None
+    vdW:           		10.4644777915 kcal / mol
+    Electrostatics:		-168.98040100000003 kcal / mol
+
+There is also, for convenience of comparing these outputs, a high-level function that calls each of
+the above functions and prepares the input to a single Pandas ``DataFrame``:
+
+```python3
+from openff.interchange.drivers.all import get_summary_data
+
+summary_data = get_summary_data(interchange)
+summary_data
+```
+                  Bond       Angle    Torsion        vdW  Electrostatics
+    OpenMM   17.309860  195.295349  13.520315  43.807859     -707.037225
+    Amber    17.310045  195.295313  13.520178  43.814430     -707.009810
+    GROMACS  17.310045  195.295578  13.520308  43.820652     -707.110899
+    LAMMPS   17.309886  195.295355  13.520323  43.783375     -707.013998
+
+This allows us to quickly look see how much each engine tends to differ in its computation of each
+component of the potential energy function:
+
+```python3
+summary_data.std()  # or .describe(), etc.
+```
+    Bond              0.000100
+    Angle             0.000121
+    Torsion           0.000069
+    vdW               0.016327
+    Electrostatics    0.046856
+    dtype: float64
+
+Are these errors on the order of 10<sup>-4</sup> - 10<sup>-2</sup> kJ/mol between engines'
+evaluation of supposedly-equivalent inputs acceptable? That's an [open
+question](https://dx.doi.org/10.1007%2Fs10822-016-9977-1) - and in this case, it's likely that the
+``Interchange``'s exports to each have some small discrepancies and how each engine is called by its
+driver is not entirely precise. Regardless, this is a useful framework for ensuring outputs to these
+engines, and engines that may be supported in the future, are consistent with each other. This also
+serves as a template for how reference energies could be distributed by force field developers:
+given a molecule topology prepared in a perscribed way, force field XYZ should provide these
+particular energies. At present, we have a decent idea of what engines _think_ these energies are
+but not what the original developers intended them to be; reference energies would enable a more
+direct route to testing engines and perhaps uncover bugs (relatively minor, but perhaps
+non-negligible) in engines themselves.
+
+
+## Python package
```

Comment:
I'm not 100% sure this section is worth the text it takes up; it serves two purposes, but I'm not sure it serves them well.
1. Try to wrap up some of the topics covered earlier and connect the functionality back to the source code
2. Provide a high-level description of the intent of each module; I have some feedback from adjacent developers that tried to poke around in the source code but got lost quickly, "you have the components, and then drivers, and then interop, and I'm not sure where stuff actually lives." Arguably this should be addressed in the documentation itself

Reply:
I think this is great to keep in the post. The contents of a blog post are "whatever you want to write about". If this section was the entire post, that would also be fine. As you say in point 2, docs are where users should go for a concise, structured explanation of specific topics, and we're free to use material from this blog post as a starting point for a page in the docs.